### ISSUE TITLE
Iterate on duration token values and implementation

### ISSUE THREAD

**Issue by johanstromqvist** [Shopify/polaris-react-archive#4762]
Follow up to the work done in #4595

**Comment by johanstromqvist** [comment]
- [x] Phase 1: #4780
- [ ] Phase 2: Ship duration values between 50ms increments. Identify what the smallest duration step should be ("primitive"), e.g. `10ms`, `16.666ms`, or `25ms`.
- [ ] Phase 3: Ship aliases that solve pattern problems, e.g. `default`, `small-enter`, `small-exit`


**Phase 1**
| Name | Value |
| ----------- | ----------- |
| duration-0 | 0ms |
| duration-50 | 50ms |
| duration-100 | 100ms |
| duration-150 | 150ms |
| duration-200 | 200ms |
| duration-250 | 250ms |
| duration-300 | 300ms |
| duration-350 | 350ms |
| duration-400 | 400ms |
| duration-450 | 450ms |
| duration-500 | 500ms |

These values are easy to understand, and I estimate that they can solve 80-90% of our use cases.

**Phase 2**
The outstanding question is how we best accommodate needs for values between the 50s, e.g. `125ms`. We can think of these as "advanced" needs, that more experienced creators might have when choreographing multiple animations or tweaking the perceived speed of a delicate detail. If we don't provide tokens for advanced use cases, creators either need to conform to the system on the expense of UX or implement the advanced solution off system. Reference discussion in [issue #4595](https://github.com/Shopify/polaris-react/issues/4595#issuecomment-976982159).

**Comment by alex-page** [comment]
I love expanding the duration scale. It removes the need for teams to hard code one-off values. I also think this would be pretty easy to adjust with find and replace if we had to update the values.

**Comment by aaronccasanova** [comment]
This looks great Johan! The separation of the three phases makes sense and promotes incremental enhancement 👍

**Comment by lgriffee** [comment]
I agree with Alex and Aaron 👍🏻 This phased approach will allow us to move quickly on the most impactful things that we are aligned on (Phase 1), while giving us more time to refine and think through the areas where we may have less alignment or certainty (Phase 2 & 3).

**Comment by alex-page** [comment]
@lgriffee this work is good to implement. So feel free to get started or hand it over to someone else.

**Comment by lgriffee** [comment]
Hoping to get `border-radius` work wrapped up today so I can take this on!

**Comment by alex-page** [comment]
I am closing this issue for now. We will go through the token values for motion intentionally in the future. Look forward to going through this issue then.

**Comment by lgriffee** [comment]
Making a note here for future work—after Phase 2 we should return to the [33ms duration value removed](https://github.com/Shopify/polaris/pull/5405/files#r840124963) from the SearchField in #5405 and assess if it should be added back. Also related to this are [questions](https://github.com/Shopify/polaris/pull/5405/files#r847644244) around consistency when focus-rings are visible in our UI and creating related documentation.

**Comment by johanstromqvist** [comment]
> I love expanding the duration scale. It removes the need for teams to hard code one-off values. I also think this would be pretty easy to adjust with find and replace if we had to update the values.

> I am closing this issue for now. We will go through the token values for motion intentionally in the future. Look forward to going through this issue then.

As we've discussed, making sure that product teams have the values they need is crucial for both our coverage mission and design quality. I'm concerned that closing this issue is a deprioritization. I would much rather see it on the roadmap. How can I help to make that happen?
